cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -195,7 +169,7 @@ func (s *initramfsMountsSuite) TestInitramfsMountsInstallModeStep3(c *C) {
 			return true, nil
 		case 2:
 			c.Check(path, Equals, filepath.Join(s.runMnt, "base"))
-			return true, nil
+			return false, nil
 		case 3:
```

Comment:
We don't need this step anymore, do we? So I think we can remove "case 3:" here?

Reply:
The diff is confusing to read.

Step3 function is gone, despite context mentioning it.

Step2 & Step 3 tests are merged and now base is not mounted before tmpfs ubuntu-data is emitted. I guess best to read previous and new tests side-by-side....

Reply:
And we do check that ubuntu-data is not mounted before offering to tmpfs mount it. So yes, case 3 is still needed, just like it was in the standalone Step3 testcase.